sdk/identity/identity/src/credentials/azurePowerShellCredential.ts

```diff
@@ -171,9 +171,22 @@ export class AzurePowerShellCredential implements TokenCredential {
 
           $result = New-Object -TypeName PSObject
           $result | Add-Member -MemberType NoteProperty -Name ExpiresOn -Value $token.ExpiresOn
-          if ($useSecureString) {
-            $result | Add-Member -MemberType NoteProperty -Name Token -Value (ConvertFrom-SecureString -AsPlainText $token.Token)
-          } else {
+
+          if ($token.Token -is [System.Security.SecureString]) {
```

Comment:
Is there a test we can add for this code path? I see .NET added one over at https://github.com/Azure/azure-sdk-for-net/pull/50654.